ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/dps-interfaces.j2

```diff
@@ -0,0 +1,44 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# j2lint: disable=V1 #}
```

Comment:
Why?

Reply:
was there for `Dps1` gone now with upper case. jsut like Vxlan cf the template. will remove now

Reply:
a542649